backend/planty/dashboard/models.py

```diff
@@ -15,6 +16,9 @@ class Instruction(models.Model):
     insolation = models.CharField(max_length=50)
     fertilizing = models.PositiveIntegerField(validators=[MinValueValidator(1)])
 
+    public = models.BooleanField(default=False)
+    num_selected = models.PositiveIntegerField(default=0)
```

Comment:
Nie wiem czy nie będzie trudne trzymanie tej wartości aktualną. Może lepiej byłoby po prostu liczyć ile użytkowników czy roślin używa tej konkretnej instrukcji? Jest to mniej wydajne może, ale bardziej odporne na błędy. Jeszcze jedną opcją jest przywiązanie trigger'a do tego pola, który będzie go zwiększał/zmniejszał przy wybraniu/usunięciu odpowiedniej instrukcji.

Reply:
Wydaje mi się, że sprawdzanie za każdym razem bazy wszystkich użytkowników czy roślin, kto ma jakie instrukcje mija się z celem. Za trigger do tego pola miał służyć właśnie ten SelectInstructionView, ale uznałem, że to będzie jakoś rozwiązane na etapie tworzenia rośliny, więc usunąłem go. Chciałem, aby wartość tego pola mogła być tylko zwiększana, a jeżeli ktoś tą instrukcję usunął ze swojej listy, to nie traciłaby na tym popularność, bo ktoś z tego wcześniej korzystał.